### ISSUE TITLE
2023.3-RC2: NVDA+CTRL+S Cannot Change Ducking Setting

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#15686]
Originally reported in [this thread](https://nvda.groups.io/g/nvda/topic/102181944?p=Created,,,100,1,0,0::recentpostdate/sticky,,,100,2,0,102181944,previd=1698260673790442665,nextid=1696005914224156139) of NVDA mailing list.

### Steps to reproduce:
Press NVDA+control+S and modify audio ducking option.

### Actual behavior:
The option is not honoured and opening the dialog again, we can see that it has not been saved in config.

### Expected behavior:
The option should be honoured and saved. Or the option should be removed from this dialog.

### Alternative solution
We may keep this issue as is in 2023.3. It is fixed anyway with the removal of this audio option from the synth dialog.
It's just a pity to have restored this option for 2023.3 at last minute, resulting at the end with a non working option.

### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.3rc2
#### Windows version:
10
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by seanbudd** [comment]
Closing as fixed in 2024.1

**Comment by fisher729** [comment]
Hi.

This is perhaps why the audio output device and audio ducking options should have been moved to the audio settings right away. This will no doubt cause confusion for persons running NVDA 2023.3. As an aside, I don't see why WASAPI support wasn't also moved to the audio settings, seeing as though it's now enabled by default.

The quality of NVDA's UI is going downhill, and this needs to be sorted.
